cli/internal/run/run.go

```diff
@@ -937,7 +937,8 @@ func (e *execContext) exec(ctx gocontext.Context, pt *nodes.PackageTask, deps da
 	}
 
 	cmd := exec.Command(e.packageManager.Command, argsactual...)
-	cmd.Dir = pt.Pkg.Dir
+	// TODO(gsoltis): I think this might be an actual bug? cmd.Dir should probably be an absolute path
```

Comment:
I agree that this is a "bug" but I don't think it matters so long as you have to run at root.

Reply:
I believe it is a real bug affecting `--cwd`. I have a follow-up PR to fix it.